config/runtime.exs

```diff
@@ -0,0 +1,43 @@
+import Config
+
+{args, _remaining_args, _errors} =
+  OptionParser.parse(System.argv(),
```

Comment:
We should use [`parse!`](https://hexdocs.pm/elixir/1.12.3/OptionParser.html#parse!/2)

Reply:
ok, will tackle in a follow up